task6/script.js

```diff
@@ -0,0 +1,47 @@
+const df = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  {
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+  },
+  {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+  }
+];
+
+const arrayPromise = () => {
+  return new Promise((resolve, reject) => {
+    resolve(listItem);
+  });
+};
+
+arrayPromise().then((value) => {
+  return new Promise((resolve, reject) => {
```

Comment:
[IMO]
You don't have to write the description of "return new Promise ((resolve, reject) => {" twice.
I thought that the process to be implemented after 3 seconds should be only in "arrayPromise".

[IMO]
This time, there is no "catch" that is the processing destination for "reject", so I think you can delete "reject" as well.

Reply:
Thank you for the review.
I didn't know that I didn't have to write reject if I didn't need it. Correct.